packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -42,52 +57,37 @@ export function DatePicker({
   activator,
   inline,
   selected,
+  disabled = false,
+  fullWidth = false,
 }: DatePickerProps) {
   const datePickerClassNames = classnames(styles.datePicker, {
     [styles.inline]: inline,
   });
+  const wrapperClassName = classnames(styles.datePickerWrapper, {
+    [styles.fullWidth]: fullWidth,
+  });
 
   const datePickerRef = useRef() as RefObject<HTMLDivElement>;
 
   return (
-    <div className={styles.datePickerWrapper} ref={datePickerRef}>
+    <div className={wrapperClassName} ref={datePickerRef}>
       <ReactDatePicker
         calendarClassName={datePickerClassNames}
         showPopperArrow={false}
         selected={selected}
         inline={inline}
+        disabled={disabled}
         onChange={handleChange}
         formatWeekDay={date => date.substr(0, 3)}
         customInput={
-          activator ? (
-            activator
-          ) : (
-            <Button
-              variation="work"
-              type="tertiary"
-              icon="calendar"
-              ariaLabel="Open Datepicker"
-            />
-          )
+          <DatePickerActivator activator={activator} fullWidth={fullWidth} />
```

Comment:
2 issues here

- full width seems to throw an error if you use an activator that doesn't support it
- activator gets passed in as an html attribute on elements that doesn't support `activator`

let's find a way to not make the console and activator show up if it doesn't need to be there.

![image](https://user-images.githubusercontent.com/15986172/143503158-a5dbb81e-ad20-4a3c-94ba-ce890737dad1.png)